sqlmesh/core/model/definition.py

```diff
@@ -1853,8 +1825,11 @@ def create_seed_model(
 def create_python_model(
     name: str,
     entrypoint: str,
+    module_path: Path,
```

Comment:
Why was this added as a positional argument? This makes the PR breaking right? Let's double check this, these API changes are a bit risky.